test/bin/common.sh

```diff
@@ -190,6 +191,22 @@ get_vm_bridge_ip() {
 # shellcheck disable=SC2034  # used elsewhere
 VM_BRIDGE_IP="$(get_vm_bridge_ip "default")"
 
+# Web server port number
+WEB_SERVER_PORT=8080
+
+# Web server URL using VM bridge IP with fallback to host name
+# shellcheck disable=SC2034  # used elsewhere
+WEB_SERVER_URL="http://${VM_BRIDGE_IP:-$(hostname)}:${WEB_SERVER_PORT}"
+
+# Mirror registry port number
+# shellcheck disable=SC2034  # used elsewhere
```

Comment:
Probably unnecessary as the variable is used couple lines below?

Reply:
I will fix it when rebasing https://github.com/openshift/microshift/pull/3991